views/index.html

```diff
@@ -4,14 +4,14 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>영화 리뷰</title>
-    <link rel="stylesheet" href="./static/styles/index.css" />
+    <link rel="stylesheet" href="/static/styles/index.css" />
   </head>
   <body>
     <div id="root"></div>
     <footer class="footer">
       <p>&copy; 우아한테크코스 All Rights Reserved.</p>
       <p>
-        <img src="../assets/images/woowacourse_logo.png" width="180" alt="" />
+        <img src="./static/images/woowacourse_logo.png" width="180" alt="" />
```

Comment:
```suggestion
        <img src="/static/images/woowacourse_logo.png" width="180" alt="" />
```

이렇게 절대 경로로 바꾸어주니 아래 경고는 뜨지 않더라고요! 
>No routes matched location "/detail/static/images/woowacourse_logo.png" 

저도 정확하게 이해하진 못했지만 상대 경로로 지정하니 앞에 /detail이 들어가게 되더라고요!

Reply:
앗 여기 수정을 빼먹었네요 감사합니다 수정했어욥~!